src-tauri/tauri.conf.json

```
@@ -29,7 +29,7 @@
         "icons/icon.icns",
         "icons/icon.ico"
       ],
-      "identifier": "com.tauri.dev",
+      "identifier": "us.austinh.simplex-desktop",
```

Is this package identifier similar to Java namespaces?

I'm not sure what function it provides, and what changing it does.

---

leaving it default and trying to build a production release fails.

---

Gotcha!

> Error You must change the bundle identifier in `tauri.conf.json > tauri > bundle > identifier`. The default value `com.tauri.dev` is not allowed as it must be unique across applications.

Why don't we pick something more canonical:

```suggestion
      "identifier": "chat.simplex.simplex-desktop",
```

---

That identifier should be fine but I would check with the Simplex-Chat owner before publishing a release with it.